composer.json

```diff
@@ -11,7 +11,7 @@
     },
     "require": {
         "govcms/govcms": "~1",
-        "govcms/scaffold-tooling": "~1",
+        "govcms/scaffold-tooling": "~2",
```

Comment:
@srowlands this also needs `"symfony/event-dispatcher": "4.3.11 as 3.4.35"` because the way this is fixed in govcms/govcms8 won't work in a root level composer.json like this one.

Reply:
I had literally just added that to some PaaS projects this morning, will tweak here too. Thanks @simesy